<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToolbarWidget</class>
 <widget class="QWidget" name="ImageToolbarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QToolButton" name="importImage_ToolButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/add_image.png</normaloff>:/icon/32x32/add_image.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="deleteImage_ToolButton">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>4</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/trashcan.png</normaloff>:/icon/32x32/trashcan.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="locked_ToolButton">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>3</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
border: none
}

QToolButton::checked{
border: none
}

QToolButton::hover{
border-image: url(:/icon/32x32/lock_hover.png);
}

</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/lock_off.png</normaloff>
     <normalon>:/icon/32x32/lock_on.png</normalon>:/icon/32x32/lock_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="setAlignment_ToolButton">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>3</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/alignment.png</normaloff>:/icon/32x32/alignment.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="aspectLocked_ToolButton">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>5</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
border: none
}

QToolButton::checked{
border: none
}
</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/lock_off.png</normaloff>
     <normalon>:/icon/32x32/lock_on.png</normalon>:/icon/32x32/lock_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="units_ToolButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>5</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
border: none
}

QToolButton::checked{
border: none
}
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/units_px_on.png</normaloff>:/icon/32x32/units_px_on.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>1</y>
     <width>3</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="visible_ToolButton">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>5</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton::indicator:checked{
image: url(:/icon/32x32/visible_on.png);
}

QToolButton::indicator:checked:hover{
image: url(:/icon/32x32/visible_hover.png);
}

QToolButton::indicator:unchecked{
image: url(:/icon/32x32/visible_off.png);
}

QToolButton::indicator:unchecked:hover{
image: url(:/icon/32x32/visible_hover.png);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/visible_off.png</normaloff>
     <normalon>:/icon/32x32/visible_on.png</normalon>:/icon/32x32/visible_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="opacity_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>8</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rotation_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>8</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>16</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="x_Label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="y_Label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>17</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="horizontal_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/horizontal_arrow.png</normaloff>:/icon/32x32/horizontal_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="vertical_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>16</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/vertical_arrow.png</normaloff>:/icon/32x32/vertical_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="rotation_Label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>7</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../vmisc/share/resources/icon.qrc">
     <normaloff>:/icon/32x32/clockwise.png</normaloff>:/icon/32x32/clockwise.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>1</y>
     <width>3</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xPosition_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>1</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <double>-16000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>16000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="yPosition_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>16</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <double>-16000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>16000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="height_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>16</y>
     <width>65</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>16000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="width_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>1</y>
     <width>65</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>16000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="yScale_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>16</y>
     <width>65</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xScale_DoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>1</y>
     <width>65</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65</width>
     <height>15</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../vmisc/share/resources/icon.qrc"/>
 </resources>
 <connections/>
</ui>
